plone/namedfile/tests/test_scaling.py

```diff
@@ -650,11 +649,19 @@ def testScaleInvalidation(self):
         self.assertEqual(scale1a.data, scale1b.data)
         self.assertEqual(scale2a.data, scale2b.data)
 
-        # Test that field modification invalidates scales
+        # Test changing _p_mtime on the field no longer invalidates a scale
         self.item.image._p_mtime = (dt + 1).millis()
         scale1b = self.scaling.scale("image", width=100, height=80)
         scale2b = self.scaling.scale("image", width=80, height=60)
         self.assertNotEqual(scale1b.data, scale2b.data)
+        self.assertEqual(scale1a.data, scale1b.data)
+        self.assertEqual(scale2a.data, scale2b.data)
+
+        # Test changing the fields modified timestamp invalidates scales
+        self.item.image.modified = (dt + 1).millis()
+        scale1b = self.scaling.scale("image", width=100, height=80)
+        scale2b = self.scaling.scale("image", width=80, height=60)
```

Comment:
Can you rename these to `scale1c` and `scale2c`?  Now you are overriding the variables  from a few lines up, which may be confusing.
Or use other, longer names.